sarracen/render.py

```diff
@@ -352,6 +355,40 @@ def render(data: 'SarracenDataFrame',  # noqa: F821
                 label = f"log ({label})"
             colorbar.ax.set_ylabel(label)
 
+    if plot_apr:
+        # now add the circles
+        apr_max = int(data.params['apr_max'])
+        ntrack = int(data.params['ntrack'])
+        # across each apr zone
+        for j in range(ntrack):
+            if (j > 0):
+                xpos = data.params['apr_centre_x_' + str(j+1)]
+                ypos = data.params['apr_centre_y_' + str(j+1)]
+                zpos = data.params['apr_centre_z_' + str(j+1)]
+            else:
+                xpos = data.params['apr_centre_x']
+                ypos = data.params['apr_centre_y']
+                zpos = data.params['apr_centre_z']  
+
+            # across the different levels in that zone         
+            for i in range(apr_max):
+                # for the radius
+                if (i > 0):
+                    rad = data.params['apr_regions_' + str(i+1)]
+                else:
+                    rad = data.params['apr_regions']
+
+                # for the line style
+                if ((i+1) == apr_max):
+                    iline = '-'
+                else:
+                    iline = ':'
```

Comment:
Not important, but the following is a condensed Pythonic way of doing this particular if/else block:

`iline = '-' if (i+1) == apr_max else ':'`